script/lib/utils.js

```diff
@@ -95,7 +96,35 @@ async function getCurrentBranch (gitDir) {
   return branch.trim();
 }
 
+function chunkFilenames (filenames, offset = 0) {
+  // Windows has a max command line length of 2047 characters, so we can't
+  // provide too many filenames without going over that. To work around that,
+  // chunk up a list of filenames such that it won't go over that limit when
+  // used as args. Use a much higher limit on other platforms which will
+  // effectively be a no-op.
+  const MAX_FILENAME_ARGS_LENGTH =
+    os.platform() === 'win32' ? 2047 - offset : 100 * 1024;
```

Comment:
I know that `100 * 1024` isn't new to this PR; but as long as we're here, it might be safer to have a chunk limit on non-win32 even Windows is the only place we're seeing an issue right now. According to termios, the largest supported length on Linux:

> The maximum line length is 4096 chars (including the terminating newline character); lines longer than 4096 chars are truncated. After 4095 characters, input processing (e.g., ISIG and ECHO* processing) continues, but any input data after 4095 characters up to (but not including) any terminating newline is discarded.

Reply:
Good info! I'll change the non-Windows limit to be 4095 to be on the safe side.